site/content/en/latest/concepts/rate-limiting.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Rate Limiting"
+---
+
+## Overview
+
+Rate limiting is a technique used to control the number of requests allowed over a given period to prevent overloading backend services. This helps ensure stable application performance and enhances security by mitigating abuse or potential denial of service.
+
+## Use Cases
+
+Use rate limiting to:
+
+- **Prevent Overload:** Protect backend services from being overwhelmed by excessive traffic.
+- **Provide Security:** Mitigate abusive behavior, such as Denial of Service (DoS) attacks.
+- **Ensure Fair Usage:** Enforce limits on requests from clients to ensure equitable resource use.
```

Comment:
one more
* To create API limits based on user entitlements.
https://gateway.envoyproxy.io/docs/tasks/traffic/global-rate-limit/